### PR TITLE
Admins Can Configure Zones in Storage Class

### PR DESCRIPTION
**Pull request by pospispa**
The PR https://github.com/kubernetes/kubernetes/pull/38505 added zones optional parameter to Storage Class for AWS and GCE provisioners.

That's why documentation needs to be updated accordingly.

Target release: 1.7

@jsafrane PTAL

> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.7 Features: set Milestone to `1.7` and Base Branch to `release-1.7`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box below to allow 
> reviewers fix problems on your patch and speed up the review process.
> Please delete this note before submitting the pull request.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3896)

### CODE REVIEW

File: docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -438,12 +438,13 @@ metadata:
...
 parameters:
   type: io1
-  zone: us-east-1d
+  zones: us-east-1d, us-east-1c
   iopsPerGB: "10"
 ```
 
 * `type`: `io1`, `gp2`, `sc1`, `st1`. See [AWS docs](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html) for details. Default: `gp2`.
-* `zone`: AWS zone. If not specified, volumes are generally round-robin-ed across all active zones where Kubernetes cluster has a node.
+* `zone`: AWS zone. If neither zone nor zones is specified, volumes are generally round-robin-ed across all active zones where Kubernetes cluster has a node. Note: zone and zones parameters must not be used at the same time.
```
Comment (chenopis) [inline docs/concepts/storage/persistent-volumes.md]: You might want to code highlight `zone` and `zones` in these descriptions. Looks good otherwise.

**Comment by chenopis** [comment]
Thanks for letting me know this is for 1.7. I'm moving this PR to the `release-1.7` branch.

File: docs/concepts/storage/persistent-volumes.md (hunk 1 above)
Comment (pospispa) [inline docs/concepts/storage/persistent-volumes.md]: Thanks for noticing. Added the code highlights.

**Comment by chenopis** [comment]
@jsafrane Sorry, this PR picked up other commits when I moved it from `master` to `release-1.7`, so the changes are just to `docs/concepts/storage/persistent-volumes.md`.

**Comment by pospispa** [comment]
@chenopis sorry, after I fixed your comments I also rebased the PR to the tip of the master. That's why this PR picked the other commits.

**Comment by chenopis** [comment]
@pospispa No worries. Thanks!

**Comment by jsafrane** [comment]
lgtm from tech side

**Pull request merged**